### PR TITLE
completed html-skeleton exercise - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-27 at 9 11 35 AM

### CODE REVIEW

File: html-skeleton/html-skeleton-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Where do you put non-visible content about the HTML document?
+DOCTYPE goes at the top
+- Where do you put visible content about the HTML document?
+between the title and body elements
+- Where do the `<head>` and `<body>` tags go in a valid HTML document?
+within the opening a nd closing html elements
+- What is the purpose of a `<!DOCTYPE>` declaration?
+
```
Comment (shawnkost) [inline html-skeleton/html-skeleton-quiz-notes.md]: Missing an answer here!

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Bryan!

**Comment by bbrown949** [comment]
still waking up apparently! fixed. :)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**